docs/deployment.md

```diff
@@ -88,3 +88,23 @@ To deploy to production server:
   ```shell
   sudo systemctl start docker-osmaxx.service
   ```
+
+## Deploying a Hotfix
+
+Follow the above steps until 5a.
```

Comment:
Please link the chapter instead of `above steps`. Maybe the chapter will be moved in future to a separate file.